backends/platform/sdl/amigaos/amigaos-main.cpp

```diff
@@ -24,13 +24,44 @@
 
 #if defined(__amigaos4__)
 
+#include "backends/fs/amigaos4/amigaos4-fs.h"
 #include "backends/platform/sdl/amigaos/amigaos.h"
 #include "backends/plugins/sdl/sdl-provider.h"
 #include "base/main.h"
 
 int main(int argc, char *argv[]) {
 
-	// Set up a stack cookie to avoid crashes due to too few stack set by users 
+	// The following will gather the application name and add the install path
+	// to a variable in AmigaOS4's ENV(ARC) system. It will be placed in AppPaths
+	// so that ScummVM can become AmiUpdate aware
+	const char *const appname = "ScummVM";
+
+	BPTR lock;
+	APTR oldwin;
+
+	// Obtain a lock to the home directory
+	if ((lock = IDOS->GetProgramDir())) {
+		TEXT progpath[2048];
+		TEXT apppath[1024] = "AppPaths";
+
+		if (IDOS->DevNameFromLock(lock,
+			progpath,
+			sizeof(progpath),
+			DN_FULLPATH)) {
+
+	// Stop any "Insert volume..." type requesters
```

Comment:
There seems to be a problem with my browser displaying those lines.

In all the cases (display and edit) all those lines show that there are tabs, not spaces.
If i place the cursor on those lines and move them right it's clearly a tab

I don't know what to change, i did it with the online editor once and if it's still wrong i'm not sure how to change it :-(